docs/source/index.rst

```diff
@@ -8,7 +8,7 @@ Contents:
     :caption: Getting Started
 
     overview
-    Installation and Configuration <install/index>
+    Install and Configure <install/index>
```

Comment:
I'm fine with that change, although (I think) it would be good to use consistent tense everywhere.